src/commands/economy/work.js

```diff
@@ -0,0 +1,35 @@
+import { EmbedBuilder, SlashCommandBuilder } from "discord.js";
+
+export default {
+  data: new SlashCommandBuilder().setName("work").setDescription("Work"),
+
+  async execute(client, interaction) {
+    const money = Math.floor(Math.random() * (1000 + 1));
+
+    const table = await client.prisma.economy.findFirst({
+      where: { uid: interaction.user.id },
+    });
+
+    await client.prisma.economy.upsert({
+      where: {
+        uid_guildId: {
+          guildId: interaction.guild.id,
+          uid: interaction.user.id,
+        },
+      },
+      create: {
+        guildId: interaction.guild.id,
+        uid: interaction.user.id,
+        wallet: String(money),
+      },
+      update: {
+        wallet: (parseInt(table?.wallet || "0") + money).toString(),
```

Comment:
`parseInt(null)` and `parseInt(undefined)` returns `NaN` so it would be better to allocate less memory
```suggestion
        wallet: (parseInt(table?.wallet) || 0 + money).toString(),
```

Reply:
I'll think about that

Reply:
> I'll think about that

There is nothing to think about, its just more performent approach that works the same (as I explained earlier)